lib/offsite_payments/integrations/sage_pay_form.rb

```diff
@@ -69,14 +69,20 @@ def cipher(action, key, payload)
       class Helper < OffsitePayments::Helper
         include Encryption
 
+        attr_reader :timestamp
+
+        def initialize(order, account, options={})
+          super
+          @timestamp = Time.now.strftime('%Y%m%d%H%M%S').split(//).last(4).join
```

Comment:
This can be
```ruby
Time.now.strftime("%Y%m%d%H%M%S")[-4..-1]
```

If it reads clearer.

Is the last 4 of a timestamp going to be unique enough? Or would this be better as just a 4 digit random number?